website/src/components/code-block.js

```diff
@@ -100,6 +100,8 @@ const CodeBlock = props => {
             ...style,
             padding: customProperties.spacingM,
             border: `1px solid ${colors.light.cards}`,
+            // seem to be an open bug -> https://github.com/stylelint/stylelint/issues/4020
```

Comment:
here is a direct link to the issue

https://github.com/stylelint/stylelint/issues/4020